### ISSUE TITLE
New behaviour - skip confirmation when executing marks

### ISSUE THREAD

**Issue by theophilusx** [djcb/mu#1974]
This is really just a comment regarding commit 04f795cbd0713c87, which changes mu4e behaviour so that when executing mu4e-mark-execute-all. This change is a change which I like. However, I was surprised to find that hitting 'x' to run the function (with no prefix) applied all the marks. The commit log seems to indicate that this would only happen if running with the prefix argument. 

This is not an issue (for me at least). However, it might be a surprise to some users and may not be the type of behaviour change others would like. When I first saw that this change was merged in, I was expecting that you would likely have to set a variable to turn on this behaviour (or turn off confirmation prompting), so was a little surprised when I first hit x and all the marks were executed without any prompt and without any prefix argument being supplied. 

I should note that I am using spacemacs and Emacs 28, so it is possible the way spacemacs has things configured has impacted on this change or Emacs 28 has.

Basically, this is not an 'issue' or 'bug report' - merely a heads up in case this is not how the change should be implemented. Personally, I think it is the type of behaviour which would best be controlled by setting a variable rather than using a prefix argument and should not be enabled by default. i.e. allow the user to 'turn on' this behaviour if they want. 

In case it is relevant, environment is 

Emacs 28.0.50 (current master HEAD)
Spacemacs (develop branch HEAD)
mu4e 1.5.11 (current master HEAD)
Ubuntu 20.10

**Comment by djcb** [comment]
Yeah, that's a bit too enthusiastic...

**Comment by djcb** [comment]
Thanks for the report!
